openmls/src/framing/plaintext.rs

```diff
@@ -27,119 +42,19 @@ use tls_codec::{Deserialize, Serialize, TlsByteVecU32, TlsDeserialize, TlsSerial
 ///
```

Comment:
This is again draft-17. We should not remove the draft version until we have the RFC. This is a good way to track what needs updating. E.g. structs without a draft version need to be checked going forward.

Reply:
In a perfect world I'd agree. But as this (and other) PRs show, it's commonly forgotten to update this. So I don't think it makes sense to have it (and won't add it myself). But you can of course add it if you want to.

Reply:
We could update the draft-16 statements, but I'm not sure that's a good way of tracking missing updates. As @franziskuskiefer said, people forget a lot of the time. Also structs don't necessarily change with a draft update, so we'd end up just changing the number in the comment in a lot of places.

Reply:
From a newcomer perspective:

If citation == `draft-17` -> Awesome. I don't need to look it up (again).

If citation < `draft-17` or None -> All bets are off. Look it up. Optional: Add citation to ease the life of collaborators.

I do not expect everyone to add these but I would happily continue to update them one citation at a time. Unless I have to fear that we start to remove them that is...

Reply:
FWIW I've updated the ones I've touched. These annotations should become more or less obsolete anyway as soon as we hit RFC compliance.

Reply:
Ah, sorry, that was a very general comment because I was afraid we might start to remove the versions -- which would (currently) be wrong in my opinion.